networking_cisco/plugins/cisco/service_plugins/cisco_router_gbp_plugin.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2016 Cisco Systems, Inc.  All rights reserved.
```

Comment:
one way to do it is to extract those common inheritances to a parent class. Then the gbp plugin class will inherit from this parent class + the gbp db class while the normal plugin class will just inherit from this parent class + the normal db class.